app/models/school_link.rb

```diff
@@ -1,14 +1,15 @@
 class SchoolLink < ApplicationRecord
   belongs_to :school
 
-  KIND_HEADER = -'header'
-  KIND_FOOTER = -'footer'
-  KIND_SOCIAL = -'social'
+  KIND_HEADER = -"header"
+  KIND_FOOTER = -"footer"
+  KIND_SOCIAL = -"social"
 
   VALID_KINDS = [KIND_HEADER, KIND_FOOTER, KIND_SOCIAL].freeze
 
   validates :kind, inclusion: VALID_KINDS
   validates :url, presence: true
+  validates_with RateLimitValidator, limit: 25, scope: :school_id
```

Comment:
Validate uniqueness scoped to school and `kind` instead.

Reply:
Each kind can have multiple links which is not possible if I add uniqueness scoped to school and kind.